tests/Microsoft.Bot.Builder.Azure.Tests/AzureBlobStorageTests.cs

```diff
@@ -44,7 +45,12 @@ protected override string ContainerName
             }
         }
 
-        public async void Dispose()
+        public Task InitializeAsync()
+        {
+            return Task.CompletedTask;
+        }
```

Comment:
`InitializeAsync()` is required by `IAsyncLifetime`. Let me know if there's a better way to implement this. I still consider myself C#-dumb.